docs/what-is-dawson.md

```diff
@@ -63,29 +63,29 @@ A defendant in a lawsuit. In our case, always the IRS.
 The case-initiating document containing the petition form and IRS notice(s) submitted by the petitioner or practitioner and recorded as an entry in the Docket Record.
 
 ### Case
-A dispute between two parties, in our case the petitioner and the IRS, that must be resolved by a judge
+A dispute between two parties, in our case the petitioner and the IRS, that must be resolved by a judge.
 
 #### Blocked
-Case is prevented from being set for trial. A blocked case will not display as eligible for upcoming trial sessions and cannot be manually added to a trial session until block is removed (See also Automatic Block)
+Case is prevented from being set for trial. A blocked case will not display as eligible for upcoming trial sessions and cannot be manually added to a trial session until block is removed.
 
-#### Automatic Blocked
+##### Automatic Block
 Case is temporarily blocked from trial due to a pending item or due date.
 
 #### Sealed
-Case that is restricted to the public; public users do not have access to view sealed cases; only parties associated with the case and internal users can view sealed case information
+Case that is restricted to the public; public users do not have access to view sealed cases; only parties associated with the case and internal users can view sealed case information.
 #### Parties  
 
-Anybody associated with the case e.g. Tax payers, Private Practititioners and IRS Respondents.
+Anybody associated with the case, e.g. Tax payers, Private Practititioners and IRS Respondents.
 
 #### Notes
-General note associated with a single case; this note can be created/edited/viewed by all internal users
+General notes associated with a single case; this note can be created/edited/viewed by all internal users
 ### Docket Entry
 A single, unique line item on the Docket Record, including external filings, court-issued documents, or can be system generated.
 
 ### Docket Record
 List of all documents filed and served in a case, as well as system generated entries generated by party or Tax Court actions in the system.
 ### Trial Session
```

Comment:
Maybe a blank line right before this heading.

Reply:
the blank lines in markdown don't have any effect in the formatting in the formatted file, but since there's so much inconsistency around this everywhere, I'm going to make an arbitrary rule in this file that all H1s and H2s have a space after them and all H3s+ don't.